test_patching_tool_version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source _liferay_common.sh
+source _test_common.sh
+
+function main {
+	test_patching_tool_version
+}
+
+function test_patching_tool_version {
+	local latest_patching_tool_version=$(./patching_tool_version.sh "4.0")
+
+	assert_equals \
+		"${latest_patching_tool_version}" \
+		$(lc_curl https://releases.liferay.com/tools/patching-tool/LATEST-4.0.txt)
```

Comment:
Nice job, @lucasmiranda0!

Can you please add tests to cover all versions of Patching Tool?

You can get an inspiration from [here](https://github.com/liferay/liferay-docker/blob/41ce6a1fac24f96578588f124eb6cca8a34b96c9/test_build_bundle_image.sh#L22-L32)